go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/RoaringBitmap/roaring v0.9.4
 	github.com/bits-and-blooms/bitset v1.2.0
 	github.com/blevesearch/bleve_index_api v1.0.3
-	github.com/blevesearch/geo v0.1.13
+	github.com/blevesearch/geo v0.1.14-0.20220806033936-256c7049d467
```

Comment:
@abhinavdangeti I am using v0.14 here so I reckon the failure isn't due to that.
Looking into it and will let you know, thanks!